wgpu-core/src/device/resource.rs

```diff
@@ -1493,6 +1493,10 @@ impl<A: HalApi> Device<A> {
             self.features
                 .contains(wgt::Features::SHADER_EARLY_DEPTH_TEST),
         );
+        caps.set(
```

Comment:
@cwfitzgerald / wgpu maintainers: I'd love to expose this as a public function in wgpu. Bevy already has a very incomplete way to automatically map wgpu features -> naga caps, but we rarely remember to update it.

Reply:
gonna make a followup for this